node/src/chain_spec.rs

```diff
@@ -285,11 +251,11 @@ const INITIAL_DHX_DAO_TREASURY_UNLOCKED_RESERVES_BALANCE: u128 = 30_000_000_000_
 
 fn mk_genesis(endowed_accounts: Vec<AccountId>, root_key: AccountId, parachain_id: ParaId) -> GenesisConfig {
```

Comment:
could we format this like the other two functions (i.e. `dev_genesis` and `testnet_genesis`) for consistency?
and also rename it to `mainnet_genesis` instead of `mk_genesis`?
i.e. 
```
fn mainnet_genesis(
    root_key: AccountId,
    initial_authorities: Vec<AuraId>,
    endowed_accounts: Vec<AccountId>,
    id: ParaId,
) -> GenesisConfig {
```
you'll have to change line `parachain_id,` to `parachain_id: id,` like the other functions

Reply:
Sent private message